docs/libcurl/opts/CURLOPT_DEBUGFUNCTION.md

```diff
@@ -51,11 +51,13 @@ Pass a pointer to your callback function, which should match the prototype
 shown above.
 
 CURLOPT_DEBUGFUNCTION(3) replaces the standard debug function used when
-CURLOPT_VERBOSE(3) is in effect. This callback receives debug
-information, as specified in the *type* argument. This function must
-return 0. The *data* pointed to by the char * passed to this function is
-not null-terminated, but is exactly of the *size* as told by the
-*size* argument.
+CURLOPT_VERBOSE(3) is in effect. This callback receives debug information, as
+specified in the *type* argument. This function must return 0. The *data*
+pointed to by the char * passed to this function is not null-terminated, but
+is exactly of the *size* as told by the *size* argument.
+
+**WARNING** this callback may receive sensitive contents from headers and
```

Comment:
I think the warning needs to be in the `CURLINFO_TEXT` section since that's the part that's unexpected

Reply:
But that's not the only kind that can get sensitive content...

Reply:
Sure, I guess there's two things that people may not know:
* headers and data may contain sensitive information like `Authorization` tokens
* that sensitive information can come to the callback via `CURLINFO_TEXT`, not just `CURLINFO_HEADER_*` and `CURLINFO_DATA_*`

It's the latter that caught me out